Automattic/vip-design-system
# Add/dark mode from valet theme

## Description

This PR supports dark mode with the new Valet-theme tokens 🥳 🎈 🚀 . We are going to release a version to include in the VIP Dashboard. Other PRs might need more changes.

Some storybook pages complain about contrast issues, but I've checked with @donaghokeeffe using Axe Devtools, and they are correct.

The Storybook is buggy when checking for dark mode contrast issues.

## Checklist


## Steps to Test

1. Pull down PR.
2. `npm run dev`.
3. Open Storybook.
4. Enable dark mode

![dark-mode-enable](https://user-images.githubusercontent.com/3402/213520890-8071c94a-9b56-47c8-8a32-3fa9b6d49fd8.jpg)

5. Navigate and check for contrast issues using Axedevtools https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd

Ps: Focus the issues on the content part of the stories. Storybook has a lot of issues with A11Y in their theme.

You can also use this URL to review: https://deploy-preview-165--vip-design-system-components.netlify.app/

The Storybook dark mode feature is buggy; you might end up having a mix of light/dark. Open incognito and activate the dark mode again.
